integration_tests/integration/incidentDetailsSubmittedEdit.cy.ts

```diff
@@ -273,10 +281,95 @@ context('Incident details (edit after completion of report)', () => {
       incidentDetailsPage.timeInputHours().type('13')
       incidentDetailsPage.timeInputMinutes().clear()
       incidentDetailsPage.timeInputMinutes().type('00')
+      incidentDetailsPage.radioButtonsDiscovery().find('input[value="Yes"]').click()
+      incidentDetailsPage.submitButton().click()
+      cy.location().should(loc => {
+        expect(loc.pathname).to.eq(`${adjudicationUrls.detailsOfOffence.urls.start(34)}`)
+      })
+    })
+
+    it('should show no appropraite hour error if DISCOVERY radio is  selected and hour is wrong', () => {
+      cy.visit(
+        `${adjudicationUrls.incidentDetails.urls.submittedEdit(
+          'G6415GD',
+          34
+        )}?referrer=${adjudicationUrls.prisonerReport.urls.report(1524455)}`
+      )
+      const incidentDetailsPage: IncidentDetails = Page.verifyOnPage(IncidentDetails)
+      incidentDetailsPage.radioButtonsDiscovery().find('input[value="No"]').click()
+      incidentDetailsPage.timeInputMinutes().clear()
+      incidentDetailsPage.timeInputMinutes().type('13')
+      incidentDetailsPage.timeInputHoursDiscovery().clear()
+      incidentDetailsPage.timeInputHoursDiscovery().type('1300')
+      incidentDetailsPage.submitButton().click()
+      incidentDetailsPage
+        .errorSummary()
+        .find('li')
+        .then($errors => {
+          expect($errors.get(0).innerText).to.contain('Enter an incident discovery hour between 00 and 23')
+        })
+    })
+
+    it('should show no error after submit if DISCOVERY radio is  selected', () => {
+      cy.visit(
+        `${adjudicationUrls.incidentDetails.urls.submittedEdit(
+          'G6415GD',
+          34
+        )}?referrer=${adjudicationUrls.prisonerReport.urls.report(1524455)}`
+      )
+      const incidentDetailsPage: IncidentDetails = Page.verifyOnPage(IncidentDetails)
+      incidentDetailsPage.timeInputMinutesDiscovery().clear()
+      incidentDetailsPage.timeInputMinutesDiscovery().type('13')
+      incidentDetailsPage.timeInputHoursDiscovery().clear()
+      incidentDetailsPage.timeInputHoursDiscovery().type('00')
+      incidentDetailsPage.radioButtonsDiscovery().find('input[value="No"]').click()
```

Comment:
The 'no' radio button is already clicked here (otherwise you wouldn't be able to see the discovery date/hours/minutes) so this line isn't required